Packs/FeedTAXII/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Removed the **filter_args** argument in the ***taxii2-get-indicators*** command as only the *indicator* filter type was supported.
+- Removed the **filter_args** integration parameter as only the *indicator* filter type was supported.
```

Comment:
I think these can be considered "breaking changes". If you feel the same please add **Breaking Changes**. If you don't think so, ignore my comment.

Reply:
Agreed 👍